Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift

```diff
@@ -160,26 +175,14 @@ final class FakespotReviewQualityCardView: UIView, Notifiable, ThemeApplicable {
 
     private lazy var adjustedRatingLabel: UILabel = .build { label in
         let text = String.Shopping.ReviewQualityCardAdjustedRatingLabel
-        let normalFont = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
-                                                                size: UX.labelFontSize)
-        let boldFont = DefaultDynamicFontHelper.preferredBoldFont(withTextStyle: .body, size: UX.labelFontSize)
-        label.attributedText = text.attributedText(boldPartsOfString: ["adjusted rating"],
-                                                   initialFont: normalFont,
-                                                   boldFont: boldFont)
+        label.attributedText = self.subject.addAttributesTo(text: text)
```

Comment:
This label doesn't have a font anymore.

Reply:
the font is set by the MarkupAttributeUtility obj which is instantiated with the base font, explicitly setting it in the label would be redundant for the labels that have bold strings